sql-composer-mysql/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Kenny Flegal <soulstompp@gmail.com>"]
 edition = '2018'
 
 [features]
+default = ["sql-composer/mysql"]
```

Comment:
A second thing here is I used the module names directly, rather than a synthetic feature name like `dbd-mysql` like we use in the external interface.  

Speaking of things I learned while making this change:
1. Any cargo dependency marked as `optional` automatically becomes a feature name, and doesn't require a declaration in Cargo.toml.  (side note: this will make my Cargo.toml feature parser trickier).
2. `error_chain` macro allows feature flags for foreign errors, but the syntax is special because it is a macro, so they are listed inline after the foreign errors.